lutris/gui/views/base.py

```diff
@@ -28,45 +28,48 @@ def popup_contextual_menu(self, view, event):
         """Contextual menu."""
         if event.button != Gdk.BUTTON_SECONDARY:
             return
-        view.current_path = view.get_path_at_pos(event.x, event.y)
-        if view.current_path:
-            view.select()
-            model = self.get_model()
-            _iter = model.get_iter(view.current_path[0])
-            if not _iter:
-                return
-            col_id = str(model.get_value(_iter, COL_ID))
-
-            game_actions = self.get_game_actions(col_id)
+        game_ids = []
+        selection = self.get_selected()
+        if selection:
+            for path in selection:
+                iterator = view.get_model().get_iter(path)
+                game_id = view.get_model().get_value(iterator, COL_ID)
+                game_ids.append(game_id)
+            game_actions = self.get_game_actions(game_ids)
             if game_actions:
                 contextual_menu = ContextualMenu(game_actions.get_game_actions())
                 contextual_menu.popup(event, game_actions)
 
-    def get_game_actions(self, game_id):
-        if self.service:
-            db_game = get_game_for_service(self.service.id, game_id)
+    def get_game_actions(self, game_ids):
+        games = []
+        for game_id in game_ids:
+            if self.service:
+                db_game = get_game_for_service(self.service.id, game_id)
 
-            if db_game:
-                game = self.get_game_by_id(db_game["id"])
+                if db_game:
+                    game = self.get_game_by_id(db_game["id"])
+                else:
+                    db_game = ServiceGameCollection.get_game(self.service.id, game_id)
+                    game = Game.create_empty_service_game(db_game, self.service)
+            elif game_id:
+                game = self.get_game_by_id(game_id)
             else:
-                db_game = ServiceGameCollection.get_game(self.service.id, game_id)
-                game = Game.create_empty_service_game(db_game, self.service)
-        elif game_id:
-            game = self.get_game_by_id(game_id)
-        else:
-            return None
-
-        return get_game_actions(game, window=self.get_toplevel())
+                return None
+            games.append(game)
+        return get_game_actions(games, window=self.get_toplevel())
 
     def get_game_by_id(self, game_id):
         application = Gio.Application.get_default()
         game = application.get_running_game_by_id(game_id) if application else None
         return game or Game(game_id)
 
     def get_selected_game_id(self):
-        selected_item = self.get_selected_item()
+        selected_item = self.get_selected()
```

Comment:
This is not a big thing, but it's better to use plural names for lists, like ```selected_items``` here. Alternately, use the neutral name you picked for the method: ```selected = self.get_selected()```.